ansible/roles/nginx/templates/node.conf.j2

```diff
@@ -40,4 +40,12 @@ server {
 
         proxy_pass   http://localhost:{{ django_port }};
     }
+
+    location /couchdb {
+        rewrite /couchdb/(.*) /$1 break;
+        proxy_pass http://localhost:5984;
```

Comment:
I see you removed `couch_listen_address` as a variable, but I don't think it hurts to have configurable options in your templates.  Ports should always have flexibility if possible.

```
http://{{ couch_db_host|default('localhost') }}:{{ couch_db_port|default('5984') }};
```